lib/collections.js

```diff
@@ -0,0 +1,6 @@
+
+Games = new Mongo.Collection("games");
```

Comment:
There is! It's actually a part of the Meteor object itself, accessible at Meteor.users.

We control the publishing here: https://github.com/brianchitester/facequalm/blob/dev/server/startup.js#L5